pkg/scheduler/cache/status_updater/concurrency.go

```diff
@@ -164,6 +198,7 @@ func (su *defaultStatusUpdater) updatePodGroup(
 	if err != nil {
 		log.StatusUpdaterLogger.Errorf("Failed to update pod group %s/%s: %v", podGroup.Namespace, podGroup.Name, err)
 	}
+
```

Comment:
Is it possible that we will want to move the pod groups to the `applied` map only on this line where we know the operation succeeded?

Reply:
Moved